server/src/controllers/song.controllers.ts

```diff
@@ -0,0 +1,99 @@
+import { Request, RequestHandler, Response } from 'express';
+import { Song }  from '../models/song.model';
+import { SongDocument } from '../types/song.types';
+
+const sendResponse = (res: Response, statusCode: number, payload: SongDocument[] | SongDocument | string ) => {
+  return res.status(statusCode).json(payload);
+};
+
+const createSong: RequestHandler = async (req: Request, res: Response): Promise<void> => {
+  const { ...toCreate } = req.body;
```

Comment:
Pls define the type here, think this is implicitly any. Some data and other vars are also implicitly any type. You can check their current types by hovering over them.

Would be better to define all the variables, and make sure none is implicitly any. Cuz once tam's new ts config pr is merged, it will throw an error for all implicitly any type :")